### DISCUSSION TITLE
I was deploy outline use docker, and version is v0.74.0. how to upgrade to v0.76.1 ?

### DISCUSSION THREAD

**Discussion by bestbugwriter** [outline/outline#6919] [Self Hosting] [answered]
deploy use this link

https://github.com/bestbugwriter/outline-deploy

**Comment by dennorske** [comment, marked as answer]
The deploy script has a hardcoded version here:
https://github.com/bestbugwriter/outline-deploy/blob/cde872b06f998e020d54700c87857366184e3eb0/outline/docker-compose.yml#L9

You may want to change that on your setup, and then run `docker compose up --force-recreate outline` in your docker compose folder.
